webui/react/src/utils/number.ts

```diff
@@ -2,6 +2,21 @@ import { isString } from './data';
 
 const PERCENT_REGEX = /^(\d+\.?\d*|\.\d+)%$/;
 
+export const findFactorOfNumber = (n: number): number[] => {
+  const abs = Math.abs(n);
+  const factors = [];
+
+  for (let i = 1; i <= Math.floor(Math.sqrt(abs)); i++) {
```

Comment:
nice! I didn't think to use `sqrt` (was thinking `abs/2` instead), but this is better.